### DISCUSSION TITLE
“长效服务增强机制”是啥？有啥作用？

### DISCUSSION THREAD

**Discussion by LisonFan** [OdysseusYuan/Mocreak#12] [Q&A] [answered]
官网和 GitHub 上都没有介绍

**Comment by OdysseusYuan** [comment, marked as answer]
感谢您的关注。长效服务增强机制是为了确保您的 Office 尽可能处于最优运行状态。如您的 Office 在后续更新过程中，涉及到数签升级、服务延期（仅限批量授权）、Office 基建服务变更，亦或者需要升级加速时，该服务会自适应联动运行。为了不影响您的计算机性能，程序安装该服务（当且仅当您主动勾选安装）后，其会以 mvp 形式留存在底层堆栈中，进而将内存开销降低至 50K（测试平均值），当 Office 事件驱动引导该服务时，会适时提升运行优先级为“后台”，帮助您增强 Office 部署效率。
